syntax = "proto3";

// protoc moto.proto --go_out=.
// protoc moto.proto --go-grpc_out=.
// python -m grpc_tools.protoc -I . --python_out=. --grpc_python_out=. moto.proto

package protoItems;
option go_package = "./proto";

service ItemService {
  rpc GetSignature(ItemText) returns (ItemSignature);
}

message ItemSignature {
  repeated uint64 Signature = 1;
}

message ItemText {
  string Text = 1;
}
